manifests/chain.pp

```diff
@@ -24,4 +26,11 @@
     ),
     order   => '01',
   }
+
+  if $log_dropped_packets {
+    concat::fragment{"${chain}-footer":
```

Comment:
Does this need an `order` parameter?

Reply:
i guess default is `10`, so it should work?  Maybe make it super obvious that it's the footer by setting it explicitly to something bigger than 10 like 100???